extensions/wikia/CuratedContent/js/CuratedContentManagmentTool.js

```diff
@@ -206,6 +211,8 @@ $(function () {
 			return result;
 		}
 
+		window._gaq.push(['_setSampleRate', '100']);
+
```

Comment:
Because we want to track 100% percent of clicks and not just a fraction. It's described in the ticket.